Changes

```diff
@@ -79,6 +79,9 @@ _______________
 - #13045: Emphasize caution about behaviour of custom block finalizers.
   (Nick Barnes)
 
+- #13078: update Format tutorial on structural boxes to mention alignment
+ questions. (Edwin Török, review by Florian Angeletti)
```

Comment:
The author and reviewer part should be on its own line (this helps changelog analysis tools).

Reply:
Thanks, I've updated the changelog and fixed the hygiene checks about line length.